openapi/tags/DrsApiPrinciples.md

```diff
@@ -104,11 +104,16 @@ DRS servers can choose to issue either hostname-based or compact identifier-base
 | Security          | Servers have full control over their own security practices. | Server operators, in addition to maintaining their own security practices, should confirm they are comfortable with the resolver registry security practices, including protection against denial of service and namespace-hijacking attacks. (See the [Appendix: Compact Identifier-Based URIs](#tag/Compact-Identifier-Based-URIs) for more information on resolver registry security.) |
 
 ## DRS Datatypes
+DRS's job is data access, period. Therefore, the DRS API supports a simple flat content model -- every `DrsObject`, like a file, represents a single opaque blob of bytes. DRS has no understanding of the meaning of objects and only provides simple domain-agnostic metadata. Understanding the semantics of specific object types is the responsibility of the applications that use DRS to fetch those objects (e.g. samtools for BAM files, DICOM viewers for DICOM objects).
 
-DRS v1 supports two types of content:
+### Atomic Objects
+DRS can be used to access individual objects of all kinds, simple or complex, large or small, stored in type-specific formats (e.g. BAM files, VCF files, CSV files). At the API level these are all the same; at the application level, DRS clients and servers are expected to agree on object semantics using non-DRS mechanisms, including but not limited to the GA4GH Data Connect API.
 
-* a *blob* is like a file — it’s a single blob of bytes, represented by a `DrsObject` without a `contents` array
-* a *bundle* is like a folder — it’s a collection of other DRS content (either blobs or bundles), represented by a `DrsObject` with a `contents` array
+### Compound Objects
+DRS can also be used to access compound objects, consisting of two or more atomic objects related to each other in a well-specified way. See the [Appendix: Compound Objects](#tag/Working-With-Compound-Objects) for suggested best practices for working with compound objects. 
+
+### [DEPRECATED] Bundles
+Previous versions of the DRS API spec included support for a *bundle* content type, which was a folder-like collection of other DRS objects (either blobs or bundles), represented by a `DrsObject` with a `contents` array. As of v1.3, bundles have been deprecated in favor of the best practices documented in the [Appendix: Compound Objects](#tag/Working-With-Compound-Objects). A future version of the API spec may remove bundle support entirely.
```

Comment:
I would like to change "A future version of the API spec may remove bundle support entirely." to "A future version of the API spec may remove bundle support entirely and/or replace bundles with a scalable approach based on the needs of our driver projects."  The reason I want to add this is I don't think we can just say "use compound objects, the semantics of which are beyond our definition of this spec" since it just means everyone does something different.  I'm hearing from drivers that the concept of bundles is useful but they don't scale (e.g. dbGaP, gen3, etc).  And until we have something better that scales, the compound object approach with manifests gives folks a scalable way to work today with the current spec (great!).  This PR gives us a best practice when scale matters.  But I don't want to cut us off from a solution that involves introduction of a scalable set endpoint for example.

Reply:
We're slated to explore alternatives during the April 2023 Connect meeting.  See the [DRS and Data Connect](https://docs.google.com/document/d/1s5aPcNQmwmin1c5F1la4O5LJDDz9VB0p/edit) session.  

Since there has been no additional comments on this since our last DRS-focused cloud meeting on 1/23/23 I'm assuming everyone is fine with moving this forward (including the small tweak above).  I'll merge into development.

If you are a stakeholder in this topic, please bring your ideas/opinions to the above GA4GH Connect session, we will target having a viable bundles replacement during that meeting.  And will work in future versions of this call to update DRS 1.4 accordingly.